pilot/pkg/bootstrap/server.go

```diff
@@ -188,13 +190,19 @@ func NewServer(args PilotArgs) (*Server, error) {
 
 	s := &Server{}
 
+	if args.Config.ControllerOptions.ResyncPeriod != 0 {
+		s.ResyncPeriod = args.Config.ControllerOptions.ResyncPeriod
+	} else {
+		s.ResyncPeriod = 30
```

Comment:
consider adding a 'const resyncPeriod = 30' at top of file

Reply:
Ok will do